<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConverterWidget</class>
 <widget class="QTabWidget" name="ConverterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="distTab">
   <attribute name="title">
    <string>Distance</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QGroupBox" name="inputBox">
        <property name="minimumSize">
         <size>
          <width>301</width>
          <height>251</height>
         </size>
        </property>
        <property name="title">
         <string>Input Distance</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="distLineEdit">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QRadioButton" name="meters1Button">
               <property name="text">
                <string>Meters (m)</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="cm1Button">
               <property name="text">
                <string>Centimeters (cm)</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="km1Button">
               <property name="text">
                <string>Kilometers (km)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="inches1Button">
               <property name="text">
                <string>Inches (in)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="feet1Button">
               <property name="text">
                <string>Feet (f)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="miles1Button">
               <property name="text">
                <string>Miles (mi)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="fathoms1Button">
               <property name="text">
                <string>Fathoms (fa)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="Fur1Button">
               <property name="text">
                <string>Furlongs (fu)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Abyssinica SIL</family>
            <pointsize>48</pointsize>
           </font>
          </property>
          <property name="text">
           <string>=</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="outputBox">
        <property name="minimumSize">
         <size>
          <width>301</width>
          <height>251</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>251</height>
         </size>
        </property>
        <property name="title">
         <string>Output Distance</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="distOutputLabel">
               <property name="font">
                <font>
                 <family>Abyssinica SIL</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="frameShape">
                <enum>QFrame::WinPanel</enum>
               </property>
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QRadioButton" name="meters2Button">
               <property name="text">
                <string>Meters (m)</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="cm2Button">
               <property name="text">
                <string>Centimeters (cm)</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="km2Button">
               <property name="text">
                <string>Kilometers (km)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="inches2Button">
               <property name="text">
                <string>Inches (in)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="feet2Button">
               <property name="text">
                <string>Feet (f)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="miles2Button">
               <property name="text">
                <string>Miles (mi)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="fathoms2Button">
               <property name="text">
                <string>Fathoms (fa)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="Fur2Button">
               <property name="text">
                <string>Furlongs (fu)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="massTab">
   <attribute name="title">
    <string>Mass</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QGroupBox" name="inputBox_2">
        <property name="minimumSize">
         <size>
          <width>301</width>
          <height>251</height>
         </size>
        </property>
        <property name="title">
         <string>Input Mass</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="massLineEdit">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QRadioButton" name="kg1Button">
               <property name="minimumSize">
                <size>
                 <width>115</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Kilograms (kg)</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="g1Button">
               <property name="text">
                <string>Grams (g)</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="lb1Button">
               <property name="text">
                <string>Pounds (lb)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="oz1Button">
               <property name="text">
                <string>Ounces (oz)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="T1Button">
               <property name="text">
                <string>Tons (T)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="st1Button">
               <property name="text">
                <string>Stones (st)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="ca1Button">
               <property name="text">
                <string>Carats (ca)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="sl1Button">
               <property name="text">
                <string>Slugs (sl)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Abyssinica SIL</family>
            <pointsize>48</pointsize>
           </font>
          </property>
          <property name="text">
           <string>=</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_10">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="outputBox_2">
        <property name="minimumSize">
         <size>
          <width>301</width>
          <height>251</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>251</height>
         </size>
        </property>
        <property name="title">
         <string>Output Mass</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <spacer name="verticalSpacer_11">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="massOutputLabel">
               <property name="font">
                <font>
                 <family>Abyssinica SIL</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="frameShape">
                <enum>QFrame::WinPanel</enum>
               </property>
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_12">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <widget class="QRadioButton" name="kg2Button">
               <property name="text">
                <string>Kilograms (kg)</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="g2Button">
               <property name="text">
                <string>Grams (g)</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="lb2Button">
               <property name="text">
                <string>Pounds (lb)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="oz2Button">
               <property name="text">
                <string>Ounces (oz)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="T2Button">
               <property name="text">
                <string>Tons (T)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="st2Button">
               <property name="text">
                <string>Stones (st)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="ca2Button">
               <property name="text">
                <string>Carats (ca)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="sl2Button">
               <property name="text">
                <string>Slugs (sl)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
